entity/user.go

```diff
@@ -0,0 +1,20 @@
+package entity
+
+import (
+	"gorm.io/gorm"
+	"time"
+)
+
+type User struct {
+	gorm.Model
+	Name     string `gorm:"type:VARCHAR(64); NOT NULL"`
+	Username string `gorm:"unique; type:VARCHAR(32); NOT NULL;"`
+	Password string `gorm:"type:VARCHAR(128); NOT NULL"`
+	Bio      string `gorm:"type:VARCHAR(256)"`
```

Comment:
same as [previous](https://github.com/bccfilkom-be/freepass-2024/pull/5/files#r1485627211) commment